index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;500;600;700&family=Poppins:wght@200;400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>Personal portfolio</title>
+  </head>
+  <body>
+    <nav class="nav">
+      <ul class="list">
+        <li><a href="#">About</a></li>
+        <li><a href="">Work</a></li>
+        <li><a href="">Contact</a></li>
+      </ul>
+    </nav>
```

Comment:
The container of the section should be a header tag. for more information you can <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header">Check this link</a>.kindly fix that 🤓

Reply:
- [ ] Kindly make sure you always explain why the changes were requested :smile: and on top of that you provide the `resources` for more info as you did :smile: